core/templates/dev/head/pages/exploration_editor/ExplorationDataService.js

```diff
@@ -87,7 +87,7 @@ oppia.factory('ExplorationDataService', [
           }
         }, function() {
           if (errorCallback) {
-            errorCallback();
+            errorCallback;
```

Comment:
No -- at this point you need to execute the function. So keep the `()`.

Reply:
Sorry, my bad. Fixed.